README.md

```diff
@@ -1,2 +1,65 @@
-# ros_qualisys
-ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package
+ros_qualisys
+============
+
+This is ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package.
+
+## Getting started
+
+### Dependencies
+
+All dependencies are listed in the [package.xml](https://github.com/Gepetto/ros-qualisys/blob/main/package.xml) file.
+
+For building the package:
+- git
+- doxygen
+- cmake
+
+For executing the package:
+- catkin + roscpp (if with ROS1)
+- ament_cmake + rclcpp (if with ROS2)
+- Python 2 or 3 (Standard Ubuntu 18.04 or higher)
+- boost (Standard Ubuntu 18.04 or higher)
+- qualisys_cpp_sdk (installable via the robotpkg ppa)
+- tf2 (ROS 1 or 2)
+- tf2_ros (ROS 1 or 2)
+- geometry_msgs (ROS 1 or 2)
+
+### Install from source
+
+You can download this package via github:
+
+    git clone --recursive git@github.com:Gepetto/ros-qualisys.git
+
+And then build the package via CMake and Make:
+
+    source /opt/openrobots/setup.bash
```

Comment:
```suggestion
    export CMAKE_PREFIX_PATH=/opt/openrobots
```

This file doesn't exist, does it ?

Reply:
Well I personnaly craete it everytime so I tend to forget it doesn't by default